packages/quantic/force-app/main/default/lwc/quanticSvg/quanticSvg.html

```diff
@@ -0,0 +1 @@
+<template></template>
```

Comment:
See the comments from [this PR](https://github.com/coveo/ui-kit/pull/990) for this file.